src/apps/js_generic/js_generic.cpp

```diff
@@ -14,7 +14,11 @@ namespace ccfapp
 
   std::vector<ccf::js::FFIPlugin> get_js_plugins()
   {
+#if !defined(DISABLE_OE)
     return {ccf::js::openenclave_plugin};
```

Comment:
The rationale for the existence of the js oe plugin was the need to verify attestation in JS apps, which was once a requirement for ACS. It would be worth checking if that's still the case.

Reply:
Thanks, I will try to understand the situation with the plugins.

Reply:
Coincidentally, that requirement has re-emerged for another internal project, so it definitely must stay.